presto-geospatial/src/test/java/com/facebook/presto/plugin/geospatial/TestBingTileFunctions.java

```diff
@@ -80,6 +82,56 @@ public void testSerialization()
         assertEquals(tile, objectMapper.readerFor(BingTile.class).readValue(json));
     }
 
+    @Test
+    void testBingTileEncoding()
+    {
+        Random random = new Random(1491);
```

Comment:
I'm not sure if this code really has to be fuzzed.

How about standard test cases:

- Min `zoom`
- Max `zoom`
- Min `x` and `y`
- Max `x` and `y`
- Several different combinations of values in between